src/Utilities/Hooks.js

```diff
@@ -0,0 +1,10 @@
+import { useFlag, useFlagsStatus } from '@unleash/proxy-client-react';
+
+export const useFeatureFlag = (flag) => {
+  const { flagsReady } = useFlagsStatus();
+  const isFlagEnabled = useFlag(flag);
+  return flagsReady ? isFlagEnabled : false;
```

Comment:
Lets update this so that it doesnt return unless its ready
```suggestion
  return flagsReady && isFlagEnabled
```
```suggestion
  return flagsReady ? isFlagEnabled : false;
```